ironfish/src/rpc/routes/wallet/getNotes.ts

```diff
@@ -68,3 +110,19 @@ router.register<typeof GetNotesRequestSchema, GetNotesResponse>(
     })
   },
 )
+
+function includeNote(note: RpcWalletNote, filter: GetNotesRequestFilter): boolean {
+  return (
+    (filter.value?.min === undefined || BigInt(note.value) >= BigInt(filter.value.min)) &&
+    (filter.value?.max === undefined || BigInt(note.value) <= BigInt(filter.value.max)) &&
+    (filter.assetId === undefined || note.assetId === filter.assetId) &&
+    (filter.assetName === undefined || note.assetName === filter.assetName) &&
```

Comment:
Do you think it'd be okay to leave out `assetName` as a filter unless there's a specific use-case for it? I'm concerned about people treating "$IRON" as our native asset rather than our native asset ID

Reply:
Agreed, removed filtering by `assetName`